tasks/marketplacesvm-push-disk-images/marketplacesvm-push-disk-images.yaml

```diff
@@ -0,0 +1,143 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: marketplacesvm-push-disk-images
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push disk images to Cloud Marketplaces
+  params:
+    - name: snapshotPath
+      type: string
+      description: |
+        Path to the JSON string of the mapped snapshot spec in the data workspace.
+        It must be processed by the "apply-mapping" task first.
+    - name: cloudMarketplacesSecret
+      type: string
+      description: Env specific secret containing the marketplaces credentials.
+    - name: concurrentLimit
+      type: string
+      description: The maximum number of images to be pulled at once.
+      default: 3
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot spec json file resides
+  steps:
+    - name: pull-and-push-images-to-marketplaces
+      image: quay.io/konflux-ci/release-service-utils:6556e8a6b031c1aad4f0472703fd121a6e1cd45d
+      env:
+        - name: CLOUD_CREDENTIALS
+          valueFrom:
+            secretKeyRef:
+              name: $(params.cloudMarketplacesSecret)
+              key: key
+      script: |
+        #!/usr/bin/env bash
+        set -ex
```

Comment:
Is there a reason this one doesn't use `set -eux`? I think it's nice to have `-u`, because ideally you shouldn't access variables that are undefined. But I'm not saying it's strictly required, so it's up to you. Also, if you've already tested this and changing this would require more testing, I understand you'd probably want to keep it as is.

Reply:
I'll patch it up to `eux`, just a sec

Reply:
@mmalina I've updated it to `eux`

Reply:
Thanks for this suggestion @mmalina !!

In fact it detected an unbound variable!

Reply:
@lslebodn I would prefer to keep it simpler and streamlined with other tasks...

Reply:
@mmalina
it is poor excuse to not use it in Konflux  due to missing short version. :-)
one can easily combine sort and long variants without any problem
`set -euo pipefail`

Reply:
It's not really an excuse, I was just trying to find explanation for why it's not used as much (not only in our repo, but also the default tekton uses for scripts is `set -eux`) ;)

I agree it's nice and I will definitely consider using it next time I create a script.

Reply:
We have it in 9 of our tasks. It isn't not used in Konflux. Really we just ask the task script and tests/mocks file to have the same setting for testing purposes. Being consistent across tasks would be nice yeah but we have many people contributing tasks and it is kind of up to them what options they want and think make sense